edge/pkg/edged/kubeclientbridge/typed/core/v1/event_bridge.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2022 The Kubernetes Authors.
```

Comment:
```suggestion
Copyright 2024 The Kubernetes Authors.
```

And the copyright format need to be adjust.